CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+
+4.2.1 (May 2020)
+----------------
+
+- Add support for time-stamps in milliseconds([#378](https://github.com/ome/omero-figure/pull/378))
```

Comment:
It seems to me the nature of the changes introduced in this PR (new APIs, new endpoints) would justify the increment of the minor version rather than the patch version.

Reply:
makes sense

Reply:
OK, I'll rename branch (new PR) to avoid confusion.